src/mug_ffi.erl

```diff
@@ -1,16 +1,25 @@
 -module(mug_ffi).
 
--export([send/2, shutdown/1, coerce/1]).
+-export([send/2, shutdown/1, coerce_tcp_message/1]).
 
 send(Socket, Packet) ->
     normalise(gen_tcp:send(Socket, Packet)).
 
 shutdown(Socket) ->
     normalise(gen_tcp:shutdown(Socket, read_write)).
 
-normalise(ok) -> {ok, nil};
-normalise({ok, T}) -> {ok, T};
-normalise({error, {timeout, _}}) -> {error, timeout};
-normalise({error, _} = E) -> E.
+normalise(ok) ->
+    {ok, nil};
+normalise({ok, T}) ->
+    {ok, T};
+normalise({error, {timeout, _}}) ->
+    {error, timeout};
+normalise({error, _} = E) ->
+    E.
```

Comment:
This was my editor auto-formatting. I can make the formatting more like it was before if we wish.